packages/e2e/nightwatch/tests/imtrFlow.js

```diff
@@ -1,9 +1,7 @@
 const { address, domain, selectors } = require("../config");
```

Comment:
I see a lot of changes in this file. Can you revert them?

Reply:
Please undo the changes or explain why you changed it.
Also there is a newline missing on the last line. Did you setup prettier/editorconfig correctly?